R/tokens.R

```diff
@@ -334,6 +343,7 @@ tokens.corpus <- function(x,
         types = NULL, 
         what = what,
         tokenizer = tokenizer,
+        concatenator = concatenator,
```

Comment:
`tokens.tokens(concatenator = "_")` is saved in the object meta and used in the downstream. This is the main change.

Reply:
The concatenator in the object meta matters a lot for `tokens_lookup()` because phrases are automatically concatenated. 

https://github.com/quanteda/quanteda/blob/d1a73cd31635c835d92a7320903903f236bde09f/R/tokens_lookup.R#L154-L155